.github/workflows/release.yml

```diff
@@ -182,7 +182,7 @@ jobs:
 
           ASSET_ID=$(curl -s -H "Authorization: token ${{ github.token }}" \
                       https://api.github.com/repos/${{ github.repository }}/releases | \
-                      jq -r "[.[] | select(.body | contains(\"- jobId: ${{ github.run_id }}\\n\"))] | first | .assets | .[] | select(.name == \"sauce-testcafe-macos.zip\") | .id | select(. != null)")
+                      jq -r "[.[] | select(.body | contains(\"- jobId: ${{ github.run_id }}\\n\"))] | first | .assets | .[] | select(.name == \"sauce-playwright-macos.zip\") | .id | select(. != null)")
```

Comment:
Nice find! But the assets are called [playwright-macos-amd64.zip](https://github.com/saucelabs/sauce-playwright-runner/releases/download/v2.25.0/playwright-macos-amd64.zip) and [playwright-windows-amd64.zip](https://github.com/saucelabs/sauce-playwright-runner/releases/download/v2.25.0/playwright-windows-amd64.zip) respectively.